src/features/user/RegisterTrees/RegisterTreesForm.tsx

```diff
@@ -229,142 +230,130 @@ export default function RegisterTrees() {
   };
 
   return ready ? (
-    <div className="profilePage">
-      <h2 className={'profilePageTitle'}>{t('me:registerTrees')}</h2>
-      <div className={styles.registerTreesPage}>
-        {!registered ? (
-          <form onSubmit={handleSubmit(submitRegisterTrees)}>
+    <StyledForm>
+      {!registered ? (
+        <>
+          <form
+            className="inputContainer"
+            onSubmit={handleSubmit(submitRegisterTrees)}
+          >
```

Comment:
@prachigarg19 `StyledForm` should be avoided here, as you end up with a nested `<form>` inside `<form>` structure, which is invalid HTML. You will see a console error over here. 

I suggest splitting the 2 scenarios (`registered` and `!registered`) into different components - you already have a `SingleContribution` component, so you can just adapt the contents within the `<form>` tag to create another component.

Reply:
Resolved in #1853